gengokit/template/NAME-service/cmd/NAME/main.gotemplate

```diff
@@ -101,6 +119,15 @@ func submain() int {
 		return 1
 	}
 
+	if len(flag.Args()) < 1 {
+		flag.Usage()
+		return 1
+	}
+
+	ctx := context.Background()
+	for i := 0; i < len(headers); i += 2 {
+		ctx = context.WithValue(ctx, headers[i], headers[i+1])
+	}
 
 	switch flag.Args()[0] {
```

Comment:
I believe that I already added that code, on line 122. Does that code cover what you mentioned in this comment?